components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1763,6 +1763,9 @@ _load_scaffolding() {
   if [[ "$(type -t _scaffolding_begin)" == "function" ]]; then
     _scaffolding_begin
   fi
+
+  build_line "Resolving Scaffolding dependencies"
+  _resolve_dependencies
```

Comment:
I think that is probably correct. I may have an issue in my testing methodology in which I am attempting to simulate a clean studio by doing `rm -rf /hab/pkgs/core/python*` to ensure the deps get downloaded. This could be introducing a red herring on my part if we assume the TDEPS are installed when the direct dependencies are already installed.